components/Hero/src/index.scss

```diff
@@ -171,6 +192,43 @@
     --denhaag-hero-content-column-start: var(--denhaag-hero-home-content-column-start-xs, 0);
     --denhaag-hero-content-column-end: var(--denhaag-hero-home-content-column-end-xs, 12);
   }
+
+  .denhaag-hero--theme-page .denhaag-hero__container {
+    padding-block-start: var(--denhaag-hero-theme-page-container-padding-block-start-s);
+  }
+
+  .denhaag-hero--theme-page .denhaag-hero__content {
+    min-height: var(--denhaag-hero-theme-page-content-min-height-s);
+    height: var(--denhaag-hero-theme-page-content-height-s);
```

Comment:
Why a height and min-height? 

When a height is set, a min-height is redundant.

Reply:
That is necessary because the height is responsive, and these have different values.